main.js

```diff
@@ -1,5 +1,43 @@
-
 import { updateLandingPageViews } from "./counter.js";
 import { initNavbarToggle } from "./toggle.js";
+window.addEventListener("load", () => {
+  const loader = document.getElementById("loader");
+  if (loader) {
+    loader.style.display = "none";
+  }
+});
+
 updateLandingPageViews();
 initNavbarToggle();
+
+const translations = {
+  pl: {
+    about: {
+      title: "O mnie",
+      intro:
+        "Cześć, jestem Michał Terczyński, Frontend Developer ze szczególnym uwzględnieniem tworzenia gier i aplikacji interaktywnych. Mam doświadczenie w firmach takich jak NetEnt, Yggdrasil Gaming i Software Mind, gdzie zdobyłem wiedzę w tworzeniu angażujących gier slotowych oraz aplikacji biznesowych z użyciem TypeScript, React, Angular i PixiJS.",
+      details:
+        "Moje umiejętności techniczne obejmują nowoczesne frameworki JavaScript, tworzenie gier za pomocą PixiJS i budowanie skalowalnych aplikacji internetowych. Pasjonuję się tworzeniem wysokiej jakości, wydajnego kodu i chętnie pracuję nad projektami komercyjnymi oraz własnymi gier.",

```

Comment:
intro:
„Cześć, jestem Michał Terczyński, Frontend Developerem ze specjalizacją w tworzeniu gier i aplikacji interaktywnych. Zdobyłem doświadczenie w takich firmach jak NetEnt, Yggdrasil Gaming i Software Mind, gdzie zdobyłem wiedzę w zakresie tworzenia angażujących gier slotowych oraz aplikacji biznesowych z wykorzystaniem TypeScript, React, Angular i PixiJS.”

details:
„Moje umiejętności techniczne obejmują nowoczesne frameworki JavaScript, tworzenie gier w PixiJS oraz budowanie skalowalnych aplikacji internetowych. Pasjonuję się tworzeniem wysokiej jakości, wydajnego kodu i chętnie podejmuję się zarówno projektów komercyjnych, jak i autorskich.”

Reply:
@skido600 Please update these translations with ones provided by me in the above comment